contracts/FastBreakV1.cdc

```diff
@@ -596,25 +601,37 @@ pub contract FastBreakV1: NonFungibleToken {
             return false
         }
 
-        pub fun points(): UInt64 {
+        view access(all) fun points(): UInt64 {
             if let fastBreak = FastBreakV1.fastBreakGameByID[self.fastBreakGameID] {
                 if let submission = fastBreak.submissions[self.mintedTo] {
                     return submission.points
                 }
             }
             return 0
         }
+
+        access(all) fun createEmptyCollection(): @{NonFungibleToken.Collection} {
+            return <- FastBreakV1.createEmptyCollection(nftType: Type<@FastBreakV1.NFT>())
+        }
+
+        view access(all) fun getViews(): [Type] {
+            return []
+        }
+
+        access(all) fun resolveView(_ view: Type): AnyStruct? {
+            return nil
+        }
     }
 
     /// The Fast Break game token collection
     ///
-    pub resource interface FastBreakNFTCollectionPublic {
-        pub fun deposit(token: @NonFungibleToken.NFT)
-        pub fun batchDeposit(tokens: @NonFungibleToken.Collection)
-        pub fun getIDs(): [UInt64]
-        pub fun borrowNFT(id: UInt64): &NonFungibleToken.NFT
-        pub fun borrowNFTSafe(id: UInt64): &NonFungibleToken.NFT?
-        pub fun borrowFastBreakNFT(id: UInt64): &FastBreakV1.NFT? {
+    access(all) resource interface FastBreakNFTCollectionPublic {
```

Comment:
Since interfaces can inherent interfaces, you could have this interface implement NFT.CollectionPublic and include additional methods specific for FastBreak